sql/moz-fx-data-shared-prod/cloudflare_derived/os_usage_v1/schema.yaml

```diff
@@ -0,0 +1,29 @@
+fields:
+  - name: dte
```

Comment:
is this supposed to be "dte"?

Reply:
Yep! Thanks for checking - I put it like that so I don't use "date" which is a reserved word in SQL